### PR TITLE
Allow variables to be set in CUSTOMIZED_SDK_URL with nightly

### PR DESCRIPTION
**Pull request by llxia**
Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
- internal/Grinder/21978
SDK_RESOURCE=nightly
CUSTOMIZED_SDK_URL=https://openj9-artifactory.osuosl.org/artifactory/ci-openj9/Build_JDK${JDK_VERSION}_${PLATFORM}_Nightly/

image

**Review by renfeiw — APPROVED** [review]
LGTM

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -373,7 +373,8 @@ def setup() {
 			setupEnv()
 
 			if (params.SDK_RESOURCE == 'nightly' && params.CUSTOMIZED_SDK_URL) {
-				CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"
+				// remove single quote to allow variables to be set in CUSTOMIZED_SDK_URL
+				CUSTOMIZED_SDK_URL_OPTION = "-c ${params.CUSTOMIZED_SDK_URL}"
 			} else if (params.CUSTOMIZED_SDK_URL) {
 				SDK_RESOURCE = "customized"
 				CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"
```
Comment (sophia-guo) [inline buildenv/jenkins/JenkinsfileBase:380]: Should this line also updated with single quote removed?

Comment (llxia) [reply buildenv/jenkins/JenkinsfileBase:380]: No, we cannot. The purpose of having a single quote is for multiple URLs (JDK, JRE, test image, etc) in CUSTOMIZED_SDK_URL value. 
This is PR is for the special case - nightly with artifactory link. In this case, only one artifactory URL is expected, so it is okay to remove the single quote.

**Review by sophia-guo — APPROVED** [review]

**Pull request merged**